py-rse/provenance.Rmd

```diff
@@ -439,11 +439,23 @@ then review the spreadsheet file and the coded response file.
 4.  This dataset is clearly in support of an article.
     What information can you find about it, and can you find a link to it?
 
+### Evaluate a project's code provenance {#provenance-ex-eval-code}
+
+The GitHub repository https://github.com/borstlab/reversephi_paper/ provides the code and data for the paper @leonhardt2017.  Browse the repository and answer the following questions:
+
+1. Where is the software environment described? What files would you need to recreate the software environment? 
+2. Where are the data processing steps described? How could you re-create the results included in the manuscript?
+3. How are the scripts and data archived?  I.e. Where can you download the version of the code and data as it was when the manuscript was published?
+
```

Comment:
```suggestion

To get a feel for the different approaches to code provenance, repeat steps 1-3 for... [TODO: List a few more papers.] 
```
This is a really good exercise. Different papers take different approaches to documenting their code, so it would be a good idea to have the reader consider a range of different papers. Maybe we should create an new issue and people can suggest papers to include in the list here (i.e. papers that do a good job with code provenance). I think we should only provide solutions for @leonhardt2017 though.

Reply:
Yes, great idea!